app/models/physical_server.rb

```diff
@@ -33,6 +33,7 @@ class PhysicalServer < ApplicationRecord
 
   virtual_column :v_availability, :type => :string, :uses => :host
   virtual_column :v_host_os, :type => :string, :uses => :host
+  virtual_delegate :ems_type, :to => "ext_management_system", :allow_nil => true
```

Comment:
@gmcculloug here's the virtual attribute definition that the external state machine depends on (for routing via messages).

Reply:
As stated above, I think this should be `emstype`.